website_upload_video/__init__.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# © 2017 Techspawn Solutions (http://techspawn.com)
```

Comment:
Don't add your copyright to non-modified files (I also don't put it on `__init__.py` files - it's a nonsense)